### PR TITLE
remove-all-laradock-built-containers-and-images

### PR DESCRIPTION
**Pull request by LarryEitel**
Remove LaraDock-built containers and images

### CODE REVIEW

**Comment by appleboy** [comment]
You just execute `docker-compose down` to remove all `laradock` container.

File: README.md [hunk 1]
```diff
@@ -1055,7 +1056,40 @@ Moving from Docker Toolbox (VirtualBox) to Docker Native (for Mac/Windows). Requ
...
+	docker images | grep laradock
+
+	echo 'dcleanlaradock completed'
+}
+# associate the above function with an alias
+# so can recall/lookup by typing 'alias'
+alias dcleanlaradock=dcleanlaradockfunction
+```
 
+**IMPORTANT!!!** This script will also delete all laradock application and data volumes!
```
Comment (appleboy) [inline README.md]: `delete all laradock container and images`

**Comment by LarryEitel** [comment]
Thanks for the input. Only thing is that `docker-compose down` did not remove laradock_* images. I could use this function to simplify the script.

Thoughts?

**Comment by LarryEitel** [comment]
I see a possible advantage of not using `docker-compose down` in that it would work on any LaraDock-based projects. You don't have to be standing in a laradock folder.

**Comment by appleboy** [comment]
@LarryEitel `docker-compose down` just remove all containers not include images.

**Comment by Mahmoudz** [comment]
Hi @LarryEitel I think this is very case specific, and won't be used by the majority of the LaraDock users..

**Pull request closed without merging**